src/main/java/teammates/common/datatransfer/FeedbackSessionResponseStatus.java

```diff
@@ -68,15 +68,19 @@ public FeedbackSessionResponseStatus() {
 
     /**
      * Returns list of students who did not respond to the feedback session.
+     * sorted by teamName > studentNamelist.
      */
     public List<String> getNoResponse() {
+        noResponse.sort(compareByTeamNameStudentName);
         return noResponse;
     }
 
     /**
      * Returns list of students who responded to the feedback session.
```

Comment:
Same here.

Reply:
This is actually part of the fix. we need the list to be sorted :)

Reply:
@mbi90 and here?

Reply:
@bqnguyen94 this and line 83 are actually the fix!